remote/analyze_handler.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python2
```

Comment:
Please add a short docstring about what the script does and for the main functions.

Reply:
Done